src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -198,10 +198,56 @@ private void SetupCommonOptionsForCommands(List<Command> commands)
                 return (int)ExitCode.ErrorInvalidARMTemplate;
             }
 
-            var analysisResult = AnalyzeTemplate(templateFilePath, parametersFilePath);
+            ExitCode exitCode;
+            if (parametersFilePath != null)
+            {
+                exitCode = AnalyzeTemplate(templateFilePath, parametersFilePath);
+            }
+            else
+            {
+                // Check if a parameters.json file is present according to naming standards here https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files#file-name, and if so use it as the parametersFile input
+                var parametersFiles = FindParameterFileForTemplate(templateFilePath);
+                int numOfFilesAnalyzed = 0;
+                bool issueReported = false;
+                var filesFailed = new List<FileInfo>();
+
+                if (parametersFiles.Count() > 0)
+                {
+                    foreach (FileInfo parametersFile in parametersFiles)
+                    {
+                        ExitCode res = AnalyzeTemplate(templateFilePath, parametersFile);
+
+                        if (res == ExitCode.Success || res == ExitCode.Violation)
+                        {
+                            numOfFilesAnalyzed++;
+                            issueReported |= res == ExitCode.Violation;
+                        }
+                        else if (res == ExitCode.ErrorAnalysis || res == ExitCode.ErrorInvalidBicepTemplate)
+                        {
+                            filesFailed.Add(templateFilePath);
+                        }
+                    }
+
+                    Console.WriteLine(Environment.NewLine + $"Analyzed {numOfFilesAnalyzed} {(numOfFilesAnalyzed == 1 ? "file" : "files")}.");
```

Comment:
```suggestion
                    Console.WriteLine(Environment.NewLine + $"Analyzed the template with {numOfFilesAnalyzed} parameter {(numOfFilesAnalyzed == 1 ? "file" : "files")}.");
```

Or something like that, that expresses that it's the same template combined with different parameter files